tools/genwqe_memcopy.c

```diff
@@ -875,11 +875,11 @@ int main(int argc, char *argv[])
 		if (total_usec) {
 			if (total_usec < 100000) {
 				kibs = ((bytes_copied * 1000000) / 1024) / total_usec;
-				VERBOSE0("%d KiB, in %ld usec, %ld KiB/sec", kib, total_usec, kibs);
+				VERBOSE0("%d KiB, in %lld usec, %lu KiB/sec", kib, total_usec, kibs);
 			} else {
 				total_msec = total_usec / 1000;	/* now msec */
 				mibs = (bytes_copied * 1000) / (1024 * 1024) / total_msec;
-				VERBOSE0("%d MiB, in %ld msec, %ld MiB/sec", mib, total_msec, mibs);
+				VERBOSE0("%d MiB, in %lld msec, %lu MiB/sec", mib, total_msec, mibs);
```

Comment:
VERBOSE0("%d MiB, in %llu msec, %lu MiB/sec", mib, (long long)total_msec, mibs);